cg/apps/lims/api.py

```diff
@@ -178,7 +178,10 @@ def capture_kit(self, lims_id: str) -> str:
             message = f"Capture kit error: {lims_sample.id} | {capture_kits}"
             raise LimsDataError(message)
 
-        return capture_kits.pop() or None
+        if len(capture_kits) == 1:
+            return capture_kits.pop()
+        else:
+            return None
```

Comment:
Would you return None or rather raise an exception?

Reply:
I was trying to preserve the previous seemingly intended behaviour to keep it patchy